newm-server/src/main/kotlin/io/newm/server/config/repo/ConfigRepository.kt

```diff
@@ -48,6 +48,7 @@ interface ConfigRepository {
         const val CONFIG_KEY_EVEARA_PARTNER_SUBSCRIPTION_ID = "eveara.partnerSubscriptionId"
         const val CONFIG_KEY_EVEARA_NEWM_EMAIL = "eveara.newmEmail"
         const val CONFIG_KEY_EVEARA_STATUS_CHECK_MINUTES = "eveara.statusCheckMinutes"
+        const val CONFIG_KEY_EVEARA_STATUS_CHECK_REFIRE = "eveara.statusCheckRefire"
```

Comment:
rename -> `eveara.statusCheckDeclinedMaxRefire`

Reply:
Alternatively, make this value a time in Days and do the math based on `eveara.statusCheckMinutes` value to determine the correct number of refires to allow. Right now, this requires the person configuring the service to do the math before setting a value.

Reply:
@AndrewWestberg  please review my latest changes to see if it matches your latest request